### PR TITLE
Added business banks to banks list

### PR DESCRIPTION
**Pull request by arifmatin**
#MOOV-2358

### CODE REVIEW

**Comment by sipsorcery** [comment]
Latest changes for this PR can be tested at https://business-dev.nofrixion.com/119.

File: packages/components/src/components/functional/CurrentAccountsList/CurrentAccountsList.tsx [hunk 1]
```diff
@@ -83,7 +83,26 @@ const CurrentAccountsMain = ({
...
+            processor: bank.processor,
+            personalInstitutionID: bank.businessInstitutionID,
+            businessInstitutionID: bank.businessInstitutionID,
+            message: bank.message,
+            messageImageUrl: bank.messageImageUrl,
+          })
+        }
+      })
+
+      setBanks(bankList)
```
[the remark this answers is not in the document]
Comment (arifmatin) [reply packages/components/src/components/functional/CurrentAccountsList/CurrentAccountsList.tsx:106]: Yeah sure. The sort is there now.

**Comment by sipsorcery** [comment]
Latest changes for this PR can be tested at https://business-dev.nofrixion.com/119.

**Review by donalnofrixion — CHANGES_REQUESTED** [review]
Can you take a look at onConnectBank in functional/CurrentAccountsList.tsx.
It might need updating/fixing

**Comment by arifmatin** [comment]
@donalnofrixion Ah I see what you mean.

**Comment by sipsorcery** [comment]
Latest changes for this PR can be tested at https://business-dev.nofrixion.com/119.

File: packages/components/src/components/functional/CurrentAccountsList/CurrentAccountsList.tsx [hunk 2]
```diff
@@ -123,7 +124,8 @@ const CurrentAccountsMain = ({
   }, [bankId, banks])
 
   const onConnectBank = async (bank: BankSettings) => {
-    // TODO: Fix this. Which one should we use?
+    // Hack: Personal and Business banks have separate records and the institution ID
+    // is set on the personalInstitutionID field for both personal and business.
```
Comment (donalnofrixion) [inline packages/components/src/components/functional/CurrentAccountsList/CurrentAccountsList.tsx:128]: Are all environments set up this way? If needs be can you create a script to update them?

**Review by donalnofrixion — APPROVED** [review]

File: packages/components/src/components/functional/CurrentAccountsList/CurrentAccountsList.tsx (hunk 2 above)
Comment (arifmatin) [reply packages/components/src/components/functional/CurrentAccountsList/CurrentAccountsList.tsx:128]: Sorry this comment may be misleading. In the database both personal and business is 1 record. I added a method that separates it to 2 records in mm4b. By the time it gets here, the records are separated, filtered and sorted. Therefore no change needs to made in onConnectBank method or the environments.

**Pull request merged**